### PR TITLE
Adding jest-enzyme dependency in travis and running travis on dev branch

### PR DESCRIPTION
**Pull request by saroup**
We need to run travis on dev branch to make sure that after merging different branches it is not broken.
Also fixing travis in dev branch with this checkin

### CODE REVIEW

**Comment by saroup** [comment]
@margaryta is added to the review. #Closed

**Comment by saroup** [comment]
@in4margaret is added to the review. #Closed

**Review by in4margaret — APPROVED** [review]
Good! Lets test on merge

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 sudo: required
 dist: trusty
 
+branches:
+  - dev
```
Comment (saroup) [inline .travis.yml:5]: this is a test comment

**Pull request merged**